benchmarks/models/hf_gpt2.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Run it at its parent folder, and check result at ../perf.
+# USAGE - ./benchmark.sh
+#   [fairseq|fairseq+fastseq|transformers|transformers+fastseq]
+#   <model>
+#   <task>
+#   <split> # train/val/test (text) or train/valid/test (binary)
+#   <batch-sizes>
+source hf.sh
+
+# MODEL - bart large cnn from transformer
+# TASK - cnn dm val full set
+
+./benchmark.sh \
+    transformers+fastseq \
+    gpt2 \
+    cnn_dm.1k/raw \
```

Comment:
one is 1k set, the other is full set.

Reply:
Oh, this is for debug, will change it back.